converter/internal/staticconvert/testdata/prom_remote_write.river

```diff
@@ -1,65 +1,41 @@
 prometheus.remote_write "test1" {
 	endpoint {
-		name = "test1-661de2"
+		name = "test1-ca06cf"
 		url  = "http://localhost:9009/api/prom/push"
 
-		queue_config {
-			capacity             = 2500
-			max_shards           = 200
-			max_samples_per_send = 500
-		}
+		queue_config { }
 
-		metadata_config {
-			max_samples_per_send = 500
-		}
```

Comment:
I believe this is the same story as for the prometheus configs, with another interesting issue: when we update prometheus version and it changes defaults, the static mode's defaults will change too.

Reply:
good catch 😄 we should make sure the static config docs reflect that or override the static defaults to what they were. 

https://grafana.com/docs/agent/next/static/configuration/

Reply:
Good point. We seem to defer to prometheus documentation and I've updated the link to point to the right version here: https://grafana.com/docs/agent/next/static/configuration/metrics-config/#metrics_instance_config 

We don't talk about defaults in staticmode, so I think we're good with the link update.